docs/openapi.yaml

```diff
@@ -2133,7 +2134,8 @@ components:
           example: '2019-06-01T00:00:00Z'
           description: |-
             The date and time the campaign ends.
-            Needs to be UTC.
+            This is a required field for some campaign types: `PERIOD_CAMPAIGN`, `EVENT_CAMPAIGN` and `ULL_FLEX_CAMPAIGN`.
```

Comment:
This is not correct 😅
eventDate is currently only used by `EVENT_CAMPAIGN`, and is illegal for other types

Reply:
Nice. Fixed.